packages/tables/src/styled/StyledHiddenCell.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { hideVisually } from 'polished';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'tables.cell';
```

Comment:
```suggestion
const COMPONENT_ID = 'tables.hidden_cell';
```

This currently conflicts with `StyledCell`. True story: we used to keep a custom linter to ensure unique `COMPONENT_ID`s, but it got too unwieldy to maintain.

Reply:
Fixed 🚀